agent/githubHook.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8
+import github3
+import json
+import os
+from getpass import getpass
+
+KEY_LABEL = 'iskakalunan'
+STATE_SUCCESS = 'success'
+STATE_PENDING = 'pending'
+
+
+class githubHook(object):
+    def __init__(self):
+        user, password = self._get_credentials()
+        g = github3.login(user, password)
+        self.hasal = g.repository('Mozilla-TWQA', 'Hasal')
+
+    def _get_credentials(self):
+        github_credential = 'github.json'
```

Comment:
The reason works for me for keeping perfherder jobs work. I'll modify this back.

For credential parts, I agree it would be better if we can use authorization token. Do you think this is a must-have for draft? @askeing